Source/ExcelDna.AddIn.Tasks/ExcelDna.AddIn.Tasks.csproj

```
@@ -1,72 +1,39 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{B5CC04D3-0507-4431-9F45-7C60BAC3047A}</ProjectGuid>
+    
     <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>ExcelDna.AddIn.Tasks</RootNamespace>
-    <AssemblyName>ExcelDna.AddIn.Tasks</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net452</TargetFramework>
```

```suggestion
    <TargetFramework>net472</TargetFramework>
```

I suggest we go `net472` here given this is library used during build time only. Both Visual Studio 2017 and 2019 require .NET Framework 4.7.2 in order to **run**, so we can expect all Excel-DNA developers to have at least `net472` on their development machines.

This should also resolve the warning on `EnvDTE` being restored as `net461`.